BUILDING.md

```diff
@@ -85,7 +85,7 @@ Now that we have a working static build, it's time to make a release tarball:
 
 If you did not clone the repository with the `--recursive` option, you will get errors until you initialize the project submodules:
```

Comment:
Should this comment also be modified?

Reply:
No seems fine by me.
You can clone the repository with the `--recursive` option, but it will be a bit slower than needed.
If you didn't, you can fix it using `git submodule update --init`.